semgrep_metrics.atd

```diff
@@ -0,0 +1,138 @@
+(* Semgrep metrics.
+ *
+ * The goal of this module is to specify the schema for the metrics
+ * data sent by the Semgrep CLI to metrics.semgrep.dev.
+ *
+ * translated originally from semgrep/cli/.../metrics.py
+ *
+ * TODO:
+ *  - at some point we should move this file to semgrep-interfaces/
```

Comment:
I guess you can update the TODO

Reply:
Add a TODO maybe that we could remove some of those python default=xxx once atdpy has been fixed